library/src/main/java/com/schibsted/spain/barista/internal/viewaction/ClickChildAction.java

```diff
@@ -1,80 +1,20 @@
 package com.schibsted.spain.barista.internal.viewaction;
 
 import android.support.annotation.IdRes;
-import android.support.test.espresso.PerformException;
-import android.support.test.espresso.UiController;
 import android.support.test.espresso.ViewAction;
-import android.support.test.espresso.util.HumanReadables;
-import android.view.View;
-import android.widget.TextView;
-import com.schibsted.spain.barista.internal.failurehandler.HelperFunctionsKt;
-import com.schibsted.spain.barista.internal.util.ViewTreeAnalyzer;
-import org.hamcrest.Matcher;
 
-import static android.support.test.espresso.matcher.ViewMatchers.hasDescendant;
-import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
-import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.action.ViewActions.click;
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
-import static org.hamcrest.Matchers.allOf;
+import static com.schibsted.spain.barista.internal.viewaction.ChildActions.performOnChildWithId;
+import static com.schibsted.spain.barista.internal.viewaction.ChildActions.performOnChildWithMatcher;
 
 public class ClickChildAction {
```

Comment:
Yeah, I saw it's no longer used, but I was afraid to remove it since it's public and as such someone could be already using it. Though if you're fine with the removal, I'll do it.

Reply:
Yes, we don't promise any api retrocompatibility for code inside the `internal` package.